context_test.go

```diff
@@ -292,7 +292,7 @@ func TestContext_Set_InvalidFlagAccessHandler(t *testing.T) {
 		},
 	}
 	c := NewContext(app, set, nil)
-	c.Set("missing", "")
+	expect(t, c.Set("missing", "") != nil, true)
```

Comment:
Quick note, this is a difference between v2/v3:

https://github.com/urfave/cli/pull/1694/files#diff-e6ce689a25eaef174c2dd51fe869fabbe04a6c6afbd416b23eda138c82e761baR285